### ISSUE TITLE
Beryl prisonner interrogation

### ISSUE THREAD

**Issue by antonius123** [dalaranwow/dalaran-wow#5682]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
During the quest in Borean Tundra to interrogate the beryl prisonner, everytime you complete this quest. WoW has a critical error and crash. I've tried it 3 times and my friend tried it too and everytime we crashed after turning the quest.
**Expected behaviour**: Tell us what should happen instead
Not crash the game.
**Steps to reproduce the problem**:

1. Interrogate the beryl prisonner.
2. Turn in the quest.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Are you running a modified client or tried without any add-ons? Quest worked fine every time I did it.

**Comment by antonius123** [comment]
I'm running the client I downloaded from dalaran wow with very basic addons. recount, ackis recipe, quartz.

**Comment by mementototem** [comment]
This quest work fine, I did it a few hour ago.

**Comment by antonius123** [comment]
I'd have to test it again, but it crashed 3 times when I completed the quest, went downstairs and sold the reward to the vendor in the tower. It crashed for my friend as well. I felt like it was a bit too much to be a coincidence, but if everyone else say it works fine, maybe it is. I don't know, I can't easily test this.

**Comment by Lothloryen** [comment]
The quest works fine, indeed. Closing it.
